### ISSUE TITLE
Security issues?

### ISSUE THREAD

**Issue by rbnpi** [genekogan/p5js-osc#11]
This is a great package, and i am using it successfully to drive a p5.js visualiser controlled by the music program Sonic-Pi (sonic-pi.net). I notice when I install it that npm (ver 6.1.0) throws up a number of warnings 6 of them high risk security issues.
Whilst I will only every be using this on a local machine, so I am not particularly worried, I wonder if any of them are addressable?
npm audit says that to fix them would break things, and indeed if you try it (using force) then that is the case.

**Comment by genekogan** [comment]
yes, this is old by now, and i'm sure there's a better way to do this, but at the moment i don't have the time to investigate. if you find a good solution, please feel free to submit a PR.
